### ISSUE TITLE
2 voice writing is switched on VHV

### ISSUE THREAD

**Issue by cdf75ubc** [polyrhythm-project/polyrhythm-website#24]
ex. T323, Bartok, bar 1, bassoons, 2 voice writing is switched on VHV

**Comment by craigsapp** [comment]
I do not see bassoon in T323:

<img width="1412" alt="Screen Shot 2020-01-18 at 21 24 48" src="https://user-images.githubusercontent.com/3487289/72675439-fcad0b00-3a38-11ea-85bf-2246e2d3ed40.png">

But I do see "Bass" :-).  @epoudrier: This should rather be "Bass." or written out full (which would be better) as "Bassoon" at the beginning of an example.  Should this be fixed in the Sibelius/MusicXML files?

The problem is in the MusicXML file: The voices are labeled incorrectly in the MusicXML data for that measure.  This probably means that the Sibelius file labeling of the voices is also incorrect.

I have fixed the Humdrum file to have the voices correctly encoded, presuming that the PDF output from Sibelius is correct:


<img width="462" alt="Screen Shot 2020-01-18 at 21 41 59" src="https://user-images.githubusercontent.com/3487289/72675588-6201fb80-3a3b-11ea-83de-7c9aa1e3b73a.png">


(I also fixed the "Bass" to be labeled as "Bassoon", but only in the Humdrum file.

Here is the Sibelius PDF rendering of the measure in question:

<img width="548" alt="Screen Shot 2020-01-18 at 21 43 55" src="https://user-images.githubusercontent.com/3487289/72675609-ad1c0e80-3a3b-11ea-88df-1dd671a16629.png">

**Comment by epoudrier** [comment]
Hi Craig,

The idea was to do editing un Humdrum and Sibelius in the next round. If Humdrum is fixed at thus point for transcription errors than there is a good chance it will be missed in the next round.

If an error is transcription Issue rather thsn a rendering error, i would propose we take note and fix it later.

Ève

**Comment by craigsapp** [comment]
That sounds good.  I am marking such problems as "Sibelius bug" and "MusicXML bug" in the issues, so they will somewhat easy to find (although one issue may refer to several examples).

Here is an example link to all issued marked as "MusicXML bug":

https://github.com/polyrhythm-project/polyrhythm-website/labels/MusicXML%20bug

**Comment by epoudrier** [comment]
Sounds good. Let’s go over what each label means next week.

My schedule is packed until Thursday noon. You teach Thursday afternoon?

**Comment by epoudrier** [comment]
Re-thinking about this, I think that if it is easy enough, letM's fix it in the Humdrum file and I will make a note on the proofreading spreadsheet. As long as you make sure I get notified of these Humdrum edits, it should not be lost.

In this case, should it not be "Bassoons"? There are two on the staff.

**Comment by cdf75ubc** [comment]
ex. T324, Bartok, horns, bars 106-107, voices are mixed up on staff from VHV.

PDF

<img width="442" alt="Screen Shot 2020-01-29 at 10 02 32 AM" src="https://user-images.githubusercontent.com/59900770/73383671-9917ae80-427e-11ea-9179-8aaae671ca46.png">

VHV

<img width="541" alt="Screen Shot 2020-01-29 at 10 03 14 AM" src="https://user-images.githubusercontent.com/59900770/73383681-9ddc6280-427e-11ea-950f-a55c9d67d88c.png">

**Comment by cdf75ubc** [comment]
@epoudrier Should we combine this issue with Layer Swapping #31 ? On Spreadsheet I listed the comment before T324 Bartok under #31 instead of creating a new column.

**Comment by epoudrier** [comment]
Craig makes the decision on combining issues. You can create a new one or
add a c comment to the existing one with screenshots if you are not sure.

**Comment by craigsapp** [comment]
Yes, this seems to be a layer swapping problem in Sibelius (but I have not checked).  And the best place to fix that would be in the Sibelius file by swapping layers.

**Comment by cdf75ubc** [comment]
ex. T597, Honegger, here there is a little visualization problem regarding the placement of 8th-note rests on the same staff with 2-voice writing so that it is clear ehich voice the rest is applied to.  PDF looks good and VHV is unclear, but for the purpose, please notice, it makes no difference.

PDF

<img width="615" alt="Screen Shot 2020-02-05 at 11 45 37 AM" src="https://user-images.githubusercontent.com/59900770/73877274-376ebb80-480d-11ea-9a83-7fda34ce4fa8.png">


VHV

<img width="744" alt="Screen Shot 2020-02-05 at 11 45 18 AM" src="https://user-images.githubusercontent.com/59900770/73877281-3d649c80-480d-11ea-8f85-13f7e0aab75a.png">
